src/components/Search/SearchPageHeader/SearchFiltersBar.tsx

```diff
@@ -0,0 +1,396 @@
+    );
+
+    /**
+     * Builds the list of all filter chips to be displayed in the
+     * filter bar
+     */
+    const filters = useMemo(() => {
+        const typeValue = typeOptions.find((option) => option.value === type) ?? null;
+        const statusValue = getStatusOptions(type, groupBy).filter((option) => status.includes(option.value));
+        const dateValue = [
+            filterFormValues.dateAfter ? `${translate('common.after')} ${DateUtils.formatToReadableString(filterFormValues.dateAfter)}` : null,
+            filterFormValues.dateBefore ? `${translate('common.before')} ${DateUtils.formatToReadableString(filterFormValues.dateBefore)}` : null,
+            filterFormValues.dateOn ? `${translate('common.on')} ${DateUtils.formatToReadableString(filterFormValues.dateOn)}` : null,
+        ].filter(Boolean) as string[];
+        const fromValue = filterFormValues.from?.map((accountID) => personalDetails?.[accountID]?.displayName ?? accountID) ?? [];
+
+        const filterList = [
+            {
+                label: translate('common.type'),
+                PopoverComponent: typeComponent,
+                value: typeValue?.translation ? translate(typeValue.translation) : null,
+            },
+            {
+                label: translate('common.status'),
+                PopoverComponent: statusComponent,
+                value: statusValue.map((option) => translate(option.translation)),
+            },
+            {
+                label: translate('common.date'),
+                PopoverComponent: datePickerComponent,
+                value: dateValue,
+            },
+            {
+                label: translate('common.from'),
+                PopoverComponent: userPickerComponent,
+                value: fromValue,
+            },
+        ];
+
+        return filterList;
+    }, [
+        type,
+        groupBy,
+        filterFormValues.from,
+        filterFormValues.dateAfter,
+        filterFormValues.dateBefore,
+        filterFormValues.dateOn,
+        translate,
+        typeComponent,
+        statusComponent,
+        datePickerComponent,
+        userPickerComponent,
+        status,
+        personalDetails,
+    ]);
+
+    if (hasErrors) {
+        return null;
+    }
+
+    if (shouldShowFiltersBarLoading) {
+        return <SearchFiltersSkeleton shouldAnimate />;
+    }
+
+    const selectionButtonText = isExportMode ? translate('search.exportAll.allMatchingItemsSelected') : translate('workspace.common.selected', {count: selectedTransactionsKeys.length});
+
+    return (
+        <View style={[shouldShowSelectedDropdown && styles.ph5, styles.mb2, styles.searchFiltersBarContainer]}>
+            {shouldShowSelectedDropdown ? (
+                <View style={styles.flexRow}>
+                    <ButtonWithDropdownMenu
+                        onPress={() => null}
+                        shouldAlwaysShowDropdownMenu
+                        buttonSize={CONST.DROPDOWN_BUTTON_SIZE.SMALL}
```

Comment:
```suggestion
                        buttonSize={CONST.DROPDOWN_BUTTON_SIZE.MEDIUM}
```
![Screenshot 2025-05-21 at 12 26 25 in the morning](https://github.com/user-attachments/assets/1b72984a-90b5-4ea9-b7cd-cd21b454d61c)